### PR TITLE
AMS jesco wuester

### PR DESCRIPTION
**Pull request by jescowuester**

### CODE REVIEW

**Comment by IronhackAMS** [comment]
Hi Jesco!,

A couple of remarks here:
- Because this assignment is about making the website responsively, pay extra attention to how elements adjust to sizes starting from small screen size to extra small screen size (width < 961 px). Now you used to separate children containers  `<div class="header-left"></div>` and `<div class="header-right"></div>` to order the elements on your screen. If I were to be a client and would look at your website on a small screen size like a mobile, these elements would be too small!  Look at the end result of the assignment and notice that the bigger sections are not stacked next to each other but below each other so that's it readable on small devices. Try to fix this either with display or position properties:)
- The email address search bar is missing;)
- Try to implement media queries when working with font-sizes so that they adjust nicely depending on screen size. The same goes for images or give them a certain width.
- On the "first and second level" of parent-containers, try to use the same padding and margins. You used margin on the `<div class="nav-container">` but not on the `<div class="header-left">`/ `<div class ="header-left">. This gives you problems with responsiveness
- Try to implement flex-wrap and flex direction (= flex-flow) to get a couple of things fixed such as for example your `<div class="icon-container>

In short make, needs a couple of adjustments for responsiveness to smaller sizes!:)

Kido
